src/lib/Resources/settings/repository/inner.yml

```
@@ -157,6 +157,7 @@ services:
         alias: 'Ibexa\Core\Repository\ProxyFactory\ProxyGenerator'
 
     Ibexa\Core\Repository\ProxyFactory\ProxyDomainMapperFactory:
+        lazy: true
```

Wasn't it supposed to be this instead? :)

```suggestion
        lazy: 'Ibexa\Core\Repository\ProxyFactory\ProxyDomainMapperInterface'
```